goodreads-scrape/script_add.py

```diff
@@ -0,0 +1,21 @@
+
+#!/usr/local/bin/python
+from olclient.openlibrary import OpenLibrary
+import olclient.common as common
+import requests
+import xml.etree.ElementTree as ET
+import sys
+
+isbn_13=str(sys.argv[1])
+API_KEY='1CSpGuaV4wRbkLdaVBYag&'
+r=requests.get("https://www.goodreads.com/search.xml?key=%sq=%s"%(API_KEY,isbn_13))
+
+root=ET.fromstring(r.content) #Parsing XML response from Goodreads
+title=(root[1][6][0][8][1]).text #Deeply attributed title of the book
+author=(root[1][6][0][8][2][1]).text #author name
+image_url=(root[1][6][0][8][3]).text
+ol=OpenLibrary()
+book=common.Book(title=title,authors=[common.Author(name=author)])
```

Comment:
variable assignment should have spaces before and after equal sign -- the keyword arguments thereafter are OK as is. Space after commas:

e.g. 

```
ol = OpenLibrary()
book = common.Book(title=title, authors=[common.Author(name=author)])
```

see PEP8 style guide: https://www.python.org/dev/peps/pep-0008/#variable-annotations